user/forms.py

```diff
@@ -5,37 +5,51 @@
 
 
 class CustomUserCreationForm(UserCreationForm):
-    """
-    This class represents a form the create a new user.
+    """This class represents a form to create a new user.
     """
     class Meta(UserCreationForm.Meta):
+        """metadata for the model User. Selects fields of the moodel.
+        """
         model = User
         fields = UserCreationForm.Meta.fields + ('email', 'first_name', 'last_name', 'organisation', 'type_of_use',)
 
 
 class CustomUserChangeForm(UserChangeForm):
-    """
-    This class represents a form to change attributes of a user. The Form is used in the admin area.
+    """This class represents a form to change attributes of a user. The Form is used in the admin area.
     """
     class Meta(UserChangeForm.Meta):
+        """metadata for the model User. Selects fields of the model.
+        """
         model = User
         fields = '__all__'
 
 
 class CustomUserEditForm(UserChangeForm):
-    """
-    This class represents a form to change attributes of a user by himself.
+    """This class represents a form to change attributes of a user by himself.
+
+    Attributes
+    ----------
+    password : str
+        None
```

Comment:
"None" is not a description ;-) Please describe the attribute password, for example the requirements of the password string